engine/src/flutter/impeller/renderer/pipeline.h

```diff
@@ -92,7 +92,8 @@ extern template class Pipeline<ComputePipelineDescriptor>;
 
 PipelineFuture<PipelineDescriptor> CreatePipelineFuture(
     const Context& context,
-    std::optional<PipelineDescriptor> desc);
+    std::optional<PipelineDescriptor> desc,
+    bool async = true);
```

Comment:
Can you add a docstring for `async`?

How is it related to the deferred pso construction?  Can we just use `context.GetFlags().lazy_shader_mode` instead of having a separate argument?

Reply:
I added a doc string that explains what it is for. Its not directly relatd to lazy shader mode